redfish_client/resource.py

```diff
@@ -149,3 +156,28 @@ def wait_for(self, stat, expected, blacklisted=None, poll_interval=3, timeout=15
     @property
     def raw(self):
         return self._content
+
+    def post(self, path, payload):
+        """
+        Perform a POST at the resource with the given payload.
+
+        Args:
+          path: Path to POST to.
+          payload: The contents of the POST payload.
+        """
+        return self._connector.post(path, payload=payload)
+
+    def delete(self, path):
+        """
+        Perform a DELETE at the resource.
+
+        Args:
+          path: The path to the resource to DELETE.
+        """
+        return self._connector.delete(path)
+
+    def refresh(self):
```

Comment:
@tadeboro What do you think about the duplication with `_refesh_cache` that @aljazkosir  introduced? They kind of do the same, with `refesh` being public.

Reply:
When I first reviewed the code, that function did not exist yet. If we have a use-case that needs the refresh functionality, we can always rename the existing function and make it part of the public API.